eva/eva_server.py

```diff
@@ -83,8 +83,8 @@ def main():
 
     # Start server
     if args.start:
-        mode = ConfigurationManager().get_value("core", "mode")
-        init_builtin_udfs(mode=mode)
+        # mode = ConfigurationManager().get_value("core", "mode")
```

Comment:
Why was this disabled?

Reply:
to make testing faster on our local environment. will enable it